tslint.json

```diff
@@ -6,9 +6,25 @@
   ],
   "jsRules": {},
   "rules": {
+    "align": [true, "arguments", "elements", "members", "parameters", "statements"],
     "array-type": [true, "generic"],
+
+    // Preferred by Prettier:
```

Comment:
My my, we are getting complex these days, aren't we?

Reply:
😉 Don’t worry, it’s all automatic! This is similar to ESLint where once we enable Prettier, we can get rid of a bunch of rules.